### ISSUE TITLE
Any way to remember window positions?

### ISSUE THREAD

**Issue by timgray** [iaincollins/icarus#61]
I have to drag the windows to the side monitors every single time I launch,  any way to make them remember where they go and even auto launch them when I start Icarus?

**Comment by iaincollins** [comment]
Yeah this is minor annoyance. To be honest, I probably won't bother as dealing with things like displays changing size or configuration (and even just multiple displays on Windows) is a pain to do, but it's a reasonable request!
